### DISCUSSION TITLE
Pull request

### DISCUSSION THREAD

**Discussion by Dhrubajyo** [community/community#63617] [New to GitHub]
I want to know how to create a new pull request in a repository.

**Comment by DhrubajyotiBiswasfounderofDribll** [comment]
**_A pull request is a way to request that a project accept changes you have made to its code repository.
Click on the New Pull Request button._**

![image](https://github.com/community/community/assets/108980710/7655475f-a51b-430f-b444-4272cdc4c601)

Then if there is a template choose it or else explore it more. You can give it labels, reviewers, and assignees.

**Reply by Dhrubajyo** [reply]
Thankyou!

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
